PyRoute/Map.py

```diff
@@ -585,14 +585,22 @@ def spinward_sector(self, doc, name):
         doc.bitmap((self.spinPos[0], self.spinPos[1] - w.size[1] / 2), w, fill=self.textFill)
 
     def trailing_sector(self, doc, name):
-        size = self.namesFont.getsize(name)
+        size = self.get_text_size(self.namesFont, name)
         size = (size[0], size[1] + 3)
         txt = Image.new('L', size, 0)
         d = ImageDraw.Draw(txt)
         d.text((0, 0), name, font=self.namesFont, fill=255)
         w = txt.rotate(-90, expand=1)
         doc.bitmap((self.trailPos[0], self.trailPos[1] - w.size[1] / 2), w, fill=self.textFill)
 
+    def get_text_size(self, font, string):
```

Comment:
The core problem here isn't Python vs PyPy, it is different versions of Pillow. According to [the documentation](https://pillow.readthedocs.io/en/stable/deprecations.html#font-size-and-offset-methods) the `getsize()` method was depreciated in 9.2 and removed in 10.0. The solution to this method is to insist on using Pillow 10.0 or above. 

One challenge from using both Python and PyPy is they have different locations for installation of libraries. That is `pip3 install ...` and `pypy pip3 install...` install into different directories, and if you are upgrading a library you have to remember to do both. 

If we're only using the length of the string it's probably better to use [ImageFont.getlength()](https://pillow.readthedocs.io/en/stable/reference/ImageFont.html#PIL.ImageFont.ImageFont.getlength)

Reply:
Your original code wasn't just using the string length, but also its height.  That's the behaviour I was trying to preserve.

Reply:
The library thing was one reason why I wrote this, as I got tired of tripping over the differences.

Reply:
@tjoneslo , I've bumped the Pillow version req, but have left the text-size retrieval encapsulated for when this _next_ changes.